mono_repo/lib/src/commands/travis.dart

```diff
@@ -211,6 +211,17 @@ String _travisYml(RootConfig configs, Map<String, String> commandsToKeys) {
         '${toYaml(configs.monoConfig.travis)}\n';
   }
 
+  var defaultBranches = configs.monoConfig.travis.containsKey('branches')
```

Comment:
nit: `branchConfig`, `branches`, something like that. The default is really the string in the else here, not this variable.